master/internal/rm/multirm/multirm.go

```diff
@@ -415,6 +415,25 @@ func (m *MultiRMRouter) VerifyNamespaceExists(namespaceName string,
 	return nil
 }
 
+// RemoveEmptyNamespace removes a namespace from our interfaces in cluster if it is no
+// longer used by any workspace.
+func (m *MultiRMRouter) RemoveEmptyNamespace(namespaceName string,
+	clusterName string,
+) error {
+	if len(clusterName) == 0 {
+		return fmt.Errorf("must specify cluster name when using multiRM")
+	}
+	rm, err := m.getRM(clusterName)
+	if err != nil {
+		return fmt.Errorf("error getting resource manager for cluster %s: %w", clusterName, err)
+	}
+	err = rm.RemoveEmptyNamespace(namespaceName, clusterName)
+	if err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
Could you add a test for the multi-RM case? Something to test the different ways it might be called and confirm the error handling works as expected?